setup.py

```diff
@@ -0,0 +1,84 @@
+"""
+setup module for pshtt
+
+Based on:
+
+- https://packaging.python.org/distributing/
+- https://github.com/pypa/sampleproject/blob/master/setup.py
+"""
+
+from setuptools import setup, find_packages
+from codecs import open
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='pshtt',
+
+    # Versions should comply with PEP440
+    version='0.1.0',
+
+    description='Scan websites for HTTPS deployment best practices',
+    long_description=long_description,
+
+    # The project's main homepage
+    url='https://github.com/dhs-ncats/pshtt',
+
+    # Author details
+    author='18F',
```

Comment:
Yeah, a lot of these details were "best guess" on my part. Feel free to edit them as you see fit.